.github/workflows/docker_update_flow_visibility.yml

```
@@ -0,0 +1,52 @@
+# Anyone with write permissions to the antrea-io/antrea Github repository can
+# trigger this workflow manually, but please check with a maintainer first. The
+# workflow will tag and push flow visibility related external images into Antrea
+# Docker hub.
+name: Manually update flow visibility Docker images
+
+on:
+  workflow_dispatch:
+    inputs:
+      ch-operator-tag:
+        description: 'Tag for clickhouse-operator and metrics-exporter Docker image'
```

```suggestion
        description: 'Tag for clickhouse-operator and metrics-exporter'
```
nit: to keep the format same as grafana and ch server

---

Thanks Shawn, I added the "Docker image" in the description for Grafana and CH server instead to keep same format.